ClavaLaraApi/src-lara-clava/clava/clava/code/ForToWhileStmt.js

```diff
@@ -0,0 +1,75 @@
+laraImport("clava.ClavaJoinPoints");
+
+/**
+ * Replaces for loop with an equivalent construct based on a while loop:
+ * ```c
+ * for (init; cond; step) {
+ *   //...
+ *   continue;
+ *   //...
+ *   //...
+ * }
+ * ```
+ * becomes
+ * ```c
+ * {
+ *   init;
+ *   while (cond) {
+ *     // ...
+ *     goto __for_loop_step_${step_label_suffix};
+ *     // ...
+ *     // ...
+ * __for_loop_step_${step_label_suffix}:
+ *     step;
+ *   }
+ * }
+ * ```
+ * @param {$loop} $forStmt For-loop joinpoint
+ * @param {number|string} stepLabelSuffix Suffix to attach to the for loop step label, added in case there are `continue` statements to account for
+ * @returns The newly created replacement joinpoint
+ */
+function ForToWhileStmt($forStmt, stepLabelSuffix) {
```

Comment:
o stepLabelSuffix should be an argument or autogenerated? Even if we maintain the argument, should be auto-generated if undefined?

Should we have a global service that manages the creation of these names? (temporary variable names, labels, etc)?